release/unstable/oraclelinux/docker/Dockerfile

```diff
@@ -51,6 +51,7 @@ RUN \
         -NoLogo \
         -NoProfile \
         -Command " \
+          \$Env:POWERSHELL_TELEMETRY_OPTOUT=1; \
```

Comment:
Is this early enought?

Reply:
i think so, this looks to be the first time that PowerShell is run as part of the prep of the image

Reply:
But it's going to run inside the PowerShell.  I'll merge and We'll see.

Reply:
now that I think about this, it's probably not correct - let me think about this for a bit